projects/lib/sort-modules.js

```diff
@@ -0,0 +1,283 @@
+function cleanForSortingNumbers( values ){
+    if( Array.isArray( values ) && values.length > 0 ) {
+        const cleanedList = [];
+        const listToClean = [...values];
+
+        for (let i = 0; i < listToClean.length; i++) {

```

Comment:
Now you also know the map method, in this case, it might make sense to use it ;-)
What do you think about it?

Reply:
Like in the 016 exercise I preferred the use of filter because the array returned by map is of the same length as the original, while filter returns one containing only the values ​​that meet the indicated condition.